Description: Fix manpage-has-errors-from-man lintian issue.
 warning: macro `1f' not defined
Author: Bas Couwenberg <sebastic@debian.org>

--- a/mfhdf/ncgen/ncgen.1
+++ b/mfhdf/ncgen/ncgen.1
@@ -327,7 +327,7 @@ are all acceptable \fIfloat\fP constants
 -2.0f
 3.14159265358979f	// will be truncated to less precision
 1.f
-.1f
+\[char46]1f
 .fi
 .RE
 .LP
